tuna/miopen/parse_miopen_args.py

```diff
@@ -85,62 +91,29 @@ def get_import_cfg_parser(
                       type=str,
                       default=None,
                       help='Specify driver cmd')
-  parser.add_argument('-f',
-                      '--file_name',
-                      type=str,
-                      dest='file_name',
-                      help='File to import')
-  parser.add_argument('-F',
-                      '--framework',
-                      dest='framework',
-                      choices=[frm.value for frm in FrameworkEnum],
-                      help='Specify framework.')
-  parser.add_argument('--fw_version',
-                      dest='fw_version',
-                      type=int,
-                      default=None,
-                      required=False,
-                      help='Specify framework version')
   parser.add_argument('-g',
                       '--gpu_count',
                       dest='gpu_count',
                       type=int,
                       default=None,
                       required=False,
                       help='Specify number of gpus the benchmark runs on')
-  parser.add_argument(
-      '--mark_recurrent',
-      dest='mark_recurrent',
-      action="store_true",
-      help='Indicate whether you want the configs to be marked as recurrent')
-  parser.add_argument('-m',
-                      '--model',
-                      dest='model',
-                      choices=[model.value for model in ModelEnum],
-                      help='Specify model')
   parser.add_argument('--md_version',
                       dest='md_version',
                       type=int,
                       default=None,
                       required=False,
                       help='Specify model version')
```

Comment:
Would be best to keep these options together as they are related - model + version